test/Microsoft.ML.Tests/Transformers/KeyToValueTests.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Data.IO;
+using Microsoft.ML.Runtime.RunTests;
+using System.IO;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.ML.Tests.Transformers
+{
+    public sealed class KeyToValueTests : TestDataPipeBase
+    {
+        public KeyToValueTests(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        public void KeyToValueWorkout()
+        {
+            string dataPath = GetDataPath("iris.txt");
+
+            var reader = new TextLoader(Env, new TextLoader.Arguments
+            {
+                Column = new[]
+                {
+                    new TextLoader.Column("ScalarString", DataKind.TX, 1),
+                    new TextLoader.Column("VectorString", DataKind.TX, new[] {new TextLoader.Range(1, 4) }),
+                    new TextLoader.Column
+                    {
+                        Name="NakedKey",
```

Comment:
>NakedKey [](start = 30, length = 8)

rename. Might not pass polycheck

Reply:
Sure

---
In reply to: [216135155](https://github.com/dotnet/machinelearning/pull/856#discussion_r216135155) [](ancestors = 216135155)